index.js

```diff
@@ -1,5 +1,3 @@
-function helloWorld() {
-  return "Hello, World"
-}
+import * as Format from './formats/formats.js';
```

Comment:
First of all, we need to keep the `module.exports` part as that's how we make our stuff importable by those who install our library.
Also, maybe we try something like this?
```
import { addComma } from "...
module.exports = { addComma };
```
Then people would be able to do this:
```
const groovy = require('groovy-ui-toolkit');
groovy.addComma(1000);
```

Reply:
ahh, ok.
I trying to think of a way where we wouldn't have to keep adding to the imports of index.js. And then people would be able to do something like: `Format.addComma(1000);`

Reply:
Yea, that's true. I kinda like being intentional about what we're actually exporting though. It's possible we may eventually have some internal "helper" type files that we don't actually want to be directly exported. There are ways to export everything from a given directory though, so we might be able to eventually organize things to do that.